src/test/java/nextstep/favorite/acceptance/FavoriteAcceptanceTest.java

```diff
@@ -1,8 +1,175 @@
 package nextstep.favorite.acceptance;
 
+import io.restassured.common.mapper.TypeRef;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.exception.ExceptionResponse;
+import nextstep.favorite.application.dto.FavoriteResponse;
+import nextstep.fixture.MemberFixture;
+import nextstep.subway.domain.request.LineRequest;
 import nextstep.utils.AcceptanceTest;
+import nextstep.utils.DatabaseCleanup;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+
+import java.util.List;
+
+import static nextstep.exception.ExceptionMessage.AUTHENTICATION_FAILED;
+import static nextstep.exception.ExceptionMessage.NOT_CONNECTED_EXCEPTION;
+import static nextstep.favorite.acceptance.FavoriteSteps.*;
+import static nextstep.subway.utils.LineTestUtil.지하철_노선_생성;
+import static nextstep.subway.utils.SectionTestUtil.지하철_구간_추가;
+import static nextstep.subway.utils.StationTestUtil.지하철역_생성;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
 
 @DisplayName("즐겨찾기 관련 기능")
 public class FavoriteAcceptanceTest extends AcceptanceTest {
+
+    @Autowired
+    private DatabaseCleanup databaseCleanup;
+
+    long 강남역, 역삼역, 선릉역, 이호선;
+    String accessToken;
+
+    @BeforeEach
+    public void setUp() {
+        databaseCleanup.execute();
```

Comment:
상위 클래스인 AcceptanceTest의 setUp 메서드를 호출하면 어떨까요?
```suggestion
        super.setUp();
```